app-backend/common/src/main/scala/UserErrorHandler.scala

```diff
@@ -5,11 +5,17 @@ import akka.http.scaladsl.model.StatusCodes
 import com.typesafe.scalalogging.LazyLogging
 import spray.json.{SerializationException, DeserializationException}
 import com.typesafe.scalalogging.LazyLogging
+import org.postgresql.util.PSQLException
+
 
 trait UserErrorHandler extends Directives
     with RollbarNotifier
     with LazyLogging {
   val userExceptionHandler = ExceptionHandler {
+    case e: PSQLException if(e.getSQLState == "23505") =>
+      logger.error(RfStackTrace(e))
+      sendError(e)
```

Comment:
Do we want to send these to Rollbar?

The case I'm imagining where that's bad is where there's a client who accidentally submits the same set of uploads twice. Presumably we'll skip 409s instead of failing upload jobs for them (?), so we'd end up with a rollbar error per scene in that case. I _think_ that's bad, since it's just a user error rather than a problem in the app, but not certain.